javaslang-gwt-example/README.md

```diff
@@ -0,0 +1,17 @@
+# GWT support for Javaslang
+
+### Running the example
+
+* Install all necessary artifacts by running the following command in the main project's directory:
```

Comment:
Chances are good that we will migrate to maven soon.

Stefan Oehme (Gradle GmbH) lives also in Kiel, he offered me to help us.